modules/machine-config-daemon-metrics.adoc

```diff
@@ -11,7 +11,7 @@ The following table describes this set of metrics.
 
 [NOTE]
```

Comment:
Consider combining the two notes to avoid having notes that are side by side. From the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions): "Keep admonitions to a minimum, and avoid placing multiple admonitions close to one another."